Polyclinic/Polyclinic.Application/Service/PatientService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using AutoMapper;
+using Polyclinic.Contracts.Appointments;
+using Polyclinic.Contracts.Patients;
+using Polyclinic.Domain.Models;
+using Polyclinic.Domain.Interfaces;
+
+namespace Polyclinic.Application.Service;
+
+/// <summary>
+/// Service for managing patients
+/// </summary>
+/// <param name="patientRepository">Patient repository</param>
+/// <param name="appointmentRepository">Appointment repository</param>
+/// <param name="mapper">Mapping profile</param>
+public class PatientService(IRepository<Patient, int> patientRepository, IRepository<Appointment, int> appointmentRepository, IMapper mapper) : IPatientService
+{
+    /// <summary>
+    /// Creates a new patient
+    /// </summary>
+    /// <param name="dto">Data for creating the patient</param>
+    /// <returns>Created patient</returns>
+    public PatientDto Create(PatientCreateUpdateDto dto)
+    {
+        var newPatient = mapper.Map<Patient>(dto);
+        var lastPatient = patientRepository.ReadAll().OrderByDescending(p => p.Id).FirstOrDefault();
```

Comment:
Для поиска последнего можно использовать LastOrDefault, но мне кажется тут более приемлемо будет получение максимального значения
```suggestion
        var lastPatientId = patientRepository.ReadAll().Max(p => p.Id);
```